lib/streamlit/cli.py

```diff
@@ -55,7 +55,7 @@ def _convert_config_option_to_click_option(config_option):
     description = config_option.description
     if config_option.deprecated:
         description += "\n {} - {}".format(
-            config_option.deprecation_text, config_option.deprecation_date
+            config_option.deprecation_text, config_option.expiration_date
```

Comment:
This is weird. Looks like `deprecation_date` doesn't exist. I assume based on the structure, it's `expiration_date`

Reply:
Yup!